static/app/components/searchQueryBuilder/valueCombobox.tsx

```diff
@@ -776,16 +778,11 @@ export function SearchQueryBuilderValueCombobox({
       }
 
       // If at the start of the input and backspace is pressed, delete the last selected value
-      if (
-        e.key === 'Backspace' &&
-        e.currentTarget.selectionStart === 0 &&
-        e.currentTarget.selectionEnd === 0 &&
-        canSelectMultipleValues
-      ) {
-        dispatch({type: 'DELETE_LAST_MULTI_SELECT_FILTER_VALUE', token});
```

Comment:
I'm not the most familiar with the new search query code, but Is it expected that this action isn't being dispatched anymore? I see onDelete was modified but this call isn't in it's new definition.